lib/virtual-net/src/ruleset.rs

```diff
@@ -0,0 +1,1388 @@
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr};
```

Comment:
Add doc comment explaining what this is for, and giving an overview of the syntax.